### PR TITLE
Annoucement for Open Science + Research Software Summer School at UIUC in July/August 

### PR DESCRIPTION
**Pull request by munkm**
I'm organizing a summer school in collaboration with URSSI on open science + research software. Here's the announcement for the school. 

A few things that should be resolved:
- I said the school was co-hosted by URSSI. Is that the correct language to use here?
- I didn't use a redirect link for the registration form. Is that preferred here? I can add it if so.  
- Much of the research software themes are the same as the existing URSSI schools. I've added language on open science. I'd like to make sure it's clear to a reader that the school covers both open science and research software.

### CODE REVIEW

File: content/blog/2024-05-20-summer-software-open-science-school.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: "Applications now open for a Summer School in Open Science + Research Software Engineering"
+date: 2024-05-20
+author: "Madicken Munk"
+---
+
+
+__Do you want your science, research, and software to be open and accessible? Do you use or develop software in your research? Do you have some basic skills would like to build on and expand them?__
```
Comment (danielskatz) [inline content/blog/2024-05-20-summer-software-open-science-school.md:8]:
```suggestion
__Do you want your science, research, and software to be open and accessible? Do you use or develop software in your research? Do you have some basic skills and would like to build on and expand them?__
```

File: content/blog/2024-05-20-summer-software-open-science-school.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+If this sounds like you, then you might be interested in the upcoming Summer School in Open Science and Research Software Engineering. 
+In July 2024, we will be hosting a five day workshop on open science and research software engineering at the University of Illinois at Urbana-Champaign.
```
Comment (danielskatz) [inline content/blog/2024-05-20-summer-software-open-science-school.md:11]:
```suggestion
In July 2024, we will be hosting a five-day workshop on open science and research software engineering at the University of Illinois Urbana-Champaign. 
```

File: content/blog/2024-05-20-summer-software-open-science-school.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+author: "Madicken Munk"
+---
+
+
+__Do you want your science, research, and software to be open and accessible? Do you use or develop software in your research? Do you have some basic skills would like to build on and expand them?__
+
+If this sounds like you, then you might be interested in the upcoming Summer School in Open Science and Research Software Engineering. 
+In July 2024, we will be hosting a five day workshop on open science and research software engineering at the University of Illinois at Urbana-Champaign. 
+
+This workshop complements previous summer and winter schools hosted by URSSI on research software engineering.
```
Comment (danielskatz) [inline content/blog/2024-05-20-summer-software-open-science-school.md:13]: maybe add links?

File: content/blog/2024-05-20-summer-software-open-science-school.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+If this sounds like you, then you might be interested in the upcoming Summer School in Open Science and Research Software Engineering. 
+In July 2024, we will be hosting a five day workshop on open science and research software engineering at the University of Illinois at Urbana-Champaign. 
+
+This workshop complements previous summer and winter schools hosted by URSSI on research software engineering. 
+At this school, students will hone their open science skills in addition to building their skillset in research software engineering.  
+
+This is aimed at early-career researchers, particularly graduate students and postdocs, who are familiar with basic skills such as interacting with the Unix shell, version control using Git, and Python programming, and would like to learn more about best practices for developing research software and leveraging their research software to practice and enhance their own open science. 
+
+## Target Audience
```
Comment (danielskatz) [inline content/blog/2024-05-20-summer-software-open-science-school.md:18]: perhaps say something about being open to all disciplines (if this is right), including humanities and areas that aren't considered "science"?

Comment (danielskatz) [reply content/blog/2024-05-20-summer-software-open-science-school.md:18]: Or maybe this is just for "science"?

File: content/blog/2024-05-20-summer-software-open-science-school.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+Ideal candidates for this workshop are science practitioners that user or develop software in their research and want to share their software in their community of practice, or are contributing to other research projects.
```
Comment (danielskatz) [inline content/blog/2024-05-20-summer-software-open-science-school.md:20]:
```suggestion
Ideal candidates for this workshop are science practitioners who use or develop software in their research and want to share their software in their community of practice, or are contributing to other research projects. 
```

File: content/blog/2024-05-20-summer-software-open-science-school.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+At this school, students will hone their open science skills in addition to building their skillset in research software engineering.  
+
+This is aimed at early-career researchers, particularly graduate students and postdocs, who are familiar with basic skills such as interacting with the Unix shell, version control using Git, and Python programming, and would like to learn more about best practices for developing research software and leveraging their research software to practice and enhance their own open science. 
+
+## Target Audience
+
+Ideal candidates for this workshop are science practitioners that user or develop software in their research and want to share their software in their community of practice, or are contributing to other research projects. 
+These practitioners want to make their research open, accessible, and reproducible by implementing open science best practices in addition to building and contributing to research software. 
+
+To get the most benefit from this workshop, we expect students to be familiar with the Unix shell, Python, and git, at the level taught at a Software Carpentry Workshop.
```
Comment (danielskatz) [inline content/blog/2024-05-20-summer-software-open-science-school.md:23]: maybe link to SC?

File: content/blog/2024-05-20-summer-software-open-science-school.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+*   Software design and modularity
+*   Collaborative software development via Git+GitHub
+*   Software testing in Python
+*   Code review
+*   Packaging and distributing Python software
+*   Documentation
+*   Software, Data, and Documentation Licensing
+*   Reproducibility
+
+The school will consist of lectures on these topics along with open hacking time to allow participants to practice the concepts covered in the lectures. To facilitate the hands-on experience, each participant will bring a project to work on throughout the course for applying these concepts.
```
Comment (danielskatz) [inline content/blog/2024-05-20-summer-software-open-science-school.md:42]:
```suggestion
The school will consist of lectures on these topics along with open hacking time to allow participants to practice the concepts covered in the lectures. To facilitate the hands-on experience, each participant should bring a project to work on throughout the course for applying these concepts.
```

Comment (danielskatz) [reply content/blog/2024-05-20-summer-software-open-science-school.md:42]: or "must"?

File: content/blog/2024-05-20-summer-software-open-science-school.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+## Logistics
+
+*   **Dates**: July 29 - August 02, 2024
+*   **Location**: Urbana-Champaign, IL - [University of Illinois at Urbana-Champaign](https://illinois.edu)
```
Comment (danielskatz) [inline content/blog/2024-05-20-summer-software-open-science-school.md:47]:
```suggestion
*   **Location**: Champaign-Urbana, IL - [University of Illinois Urbana-Champaign](https://illinois.edu)
```

File: content/blog/2024-05-20-summer-software-open-science-school.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+*   **Cost**: Free (supported by a grant from the NASA Transform to Open Science Training call)
```
Comment (danielskatz) [inline content/blog/2024-05-20-summer-software-open-science-school.md:48]: again, maybe add a link?

File: content/blog/2024-05-20-summer-software-open-science-school.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+*   Travel support is available for non-local participants
+
+## Important Dates
+
+*   **Registration deadline**: 06 June 2024
+*   **Notification**: June 10
```
Comment (danielskatz) [inline content/blog/2024-05-20-summer-software-open-science-school.md:54]:
```suggestion
*   **Notification**: 10 June
```

Comment (danielskatz) [reply content/blog/2024-05-20-summer-software-open-science-school.md:54]: or consistently the other way in this line and previous one

File: content/blog/2024-05-20-summer-software-open-science-school.md (hunk 8 above)
Comment (munkm) [reply content/blog/2024-05-20-summer-software-open-science-school.md:47]: Do you think switching the order of the cities is confusing? Especially if it's the other order in the school name?

File: content/blog/2024-05-20-summer-software-open-science-school.md (hunk 7 above)
Comment (munkm) [reply content/blog/2024-05-20-summer-software-open-science-school.md:42]: What do you think about "is strongly encouraged to"?

File: content/blog/2024-05-20-summer-software-open-science-school.md (hunk 8 above)
Comment (danielskatz) [reply content/blog/2024-05-20-summer-software-open-science-school.md:47]: yes, but it's not our fault :)

File: content/blog/2024-05-20-summer-software-open-science-school.md (hunk 7 above)
Comment (danielskatz) [reply content/blog/2024-05-20-summer-software-open-science-school.md:42]: sure. Maybe also say this earlier, such as by line 16, to emphasize it

File: content/blog/2024-05-20-summer-software-open-science-school.md (hunk 4 above)
Comment (munkm) [reply content/blog/2024-05-20-summer-software-open-science-school.md:22]: Thank you so much for pointing this out! I added language saying that all domains are welcome and that we are targeting people who use or develop software in their research.

File: content/blog/2024-05-20-summer-software-open-science-school.md (hunk 8 above)
Comment (munkm) [reply content/blog/2024-05-20-summer-software-open-science-school.md]: Ok! I changed it to Urbana, as all of the lessons will be held on the Urbana side of Wright St. 🙂

**Comment by munkm** [comment]
Thank you so much for your thorough review @danielskatz ! This post is significantly improved with your suggestions. 

I've done the following:
- Made sure dates are consistently stated 
- Added links to sw carpentry, the previous URSSI workshops, and to the TOPS-T announcement blog post 
- Added language about the target audience
- Added enhancing language about the project + collaboration aspect of the workshops.

**Comment by munkm** [comment]
Here's a preview for the blog post: https://deploy-preview-52--urssi.netlify.app/blog/2024/05/20/applications-now-open-for-a-summer-school-in-open-science--research-software-engineering/

**Review by danielskatz — APPROVED** [review]
lgtm

File: content/blog/2024-05-20-summer-software-open-science-school.md [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+
+
+__Do you want your science, research, and software to be open and accessible? Do you use or develop software in your research? Do you have some basic skills and would like to build on and expand them?__
+
+If this sounds like you, then you might be interested in the upcoming Summer School in Open Science and Research Software Engineering. 
+In July 2024, we will be hosting a five-day workshop on open science and research software engineering at the University of Illinois Urbana-Champaign. 
+
+This workshop complements previous [summer](https://urssi.us/blog/2024/02/29/applications-now-open-for-the-2024-urssi-summer-school-in-research-software-engineering/) and [winter](https://urssi.us/blog/2023/10/31/applications-now-open-for-the-2024-urssi-winter-school-in-research-software-engineering/) schools hosted by URSSI on research software engineering. 
+At this school, students will hone their open science skills in addition to building their skillset in research software engineering. 
+To that end, we strongly encourage attendees to consider a particular software project to apply learning principles to during the school.
```
Comment (kyleniemeyer) [inline content/blog/2024-05-20-summer-software-open-science-school.md:15]: The existing software lessons sort of rely on people coming with something to work on - so you may want to make bringing a project (or multiple things!) mandatory and not just strongly suggested

File: content/blog/2024-05-20-summer-software-open-science-school.md [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+*   Software design and modularity
+*   Collaborative software development via Git+GitHub
+*   Software testing in Python
+*   Code review
+*   Packaging and distributing Python software
+*   Documentation
+*   Software, Data, and Documentation Licensing
+*   Reproducibility
+
+The school will consist of lectures on these topics along with open hacking time to allow participants to practice the concepts covered in the lectures. To facilitate the hands-on experience, each participant is strongly encouraged to bring a project to work on throughout the course for applying these concepts.
```
Comment (kyleniemeyer) [inline content/blog/2024-05-20-summer-software-open-science-school.md:46]: same comment as above - I recommend making bringing a project or things to work on mandatory (and part of the application)

**Review by kyleniemeyer — APPROVED** [review]

**Pull request merged**